packages/kolibri-common/components/Cards/AccessibleFolderCard.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+
+  <KCard
+    :to="to"
+    :headingLevel="headingLevel"
+    layout="horizontal"
+    thumbnailDisplay="large"
+    :title="contentNode.title"
+    :thumbnailSrc="thumbnailSrc"
+    thumbnailScaleType="centerInside"
+    thumbnailAlign="right"
+    :style="{ height: '172px', margin: '16px 0 16px 0' }"
+  >
+    <template #thumbnailPlaceholder>
+      <div class="default-folder-icon">
+        <KIcon
```

Comment:
I see. I will try and fix it asap

Reply:
@marcellamaki @AllanOXDi  for context, the placeholder element, here icon, is not displayed only when there is no thumbnail image but also while it's loading. This is a progressive loading experience, important particularly on slower networks.

The bug here is caused by `KCard` not detecting that the image has finished loading. I recently fixed this and it will be available in my upcoming KDS PR.

However @AllanOXDi on Kolibri side, it may be helpful to double-check how the icon looks like on smaller resolutions when the thumbnail image is not available. It shouldn't overflow the area in this way and Kolibri needs to configure this.

Reply:
@AllanOXDi so you can just fix the icon placement and size here if needed, but please don't add any logic regarding whether it's present or no.

Reply:
I tested it with this PR https://github.com/learningequality/kolibri-design-system/pull/785 and it seems fixed